README.md

```diff
@@ -16,7 +16,10 @@ out the pipeline in a maintainable way.
 ## 🚧 A work in-progress 🚧
 
 Prior to version 1.0, minor version updates are susceptible to breaking changes.
+You may also be interested in [Bazooka], an application which is testing out the
```

Comment:
Is it worth noting that Bazooka is an internal project? It's possible folks in the community would be confused why they got a 404 when going here.

Reply:
very good reminder about our users here